src/test/java/RestAssuredTests/TestDelete.java

```diff
@@ -0,0 +1,29 @@
+package RestAssuredTests;
```

Comment:
I didn't get what you meant by an error in the RestAssuredTests package, the tests work, and the classes in the package don't use classes related to retrofit